javagen/src/test/java/com/azure/autorest/JavagenUnitTests.java

```diff
@@ -31,4 +31,23 @@ public void generateTest(){
         System.out.println(javaPackage.getJavaFiles().size());
     }
 
+    @Test
+    public void generateWebpubsubTest(){
+        MockUnitJavagen javagen = new MockUnitJavagen();
+        // SOURCE SWAGGER URL: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json
+        // It's generated by preprocessor. It can be found under preprocessor module after calling autorest for generation.
+        String fileName = "webpubsub-code-model-processed-no-tags.yaml";
```

Comment:
Reason about needing a new test input?

Reply:
I wanted to test the javadoc from various swagger inputs. It's kind of like an experiment.